openfisca_aotearoa/tests/parental_leave/parental_leave.yaml

```diff
@@ -0,0 +1,24 @@
+---
+
+- name: "Paid Parental Leave"
+  period: 2018-08
+  absolute_error_margin: 0
+  persons:
+    - id: "A Primary Carer"
```

Comment:
can we have tests for people who are not eligible. e.g.:

```
      is_nz_citizen: true
      parental_leave__is_primary_carer: true
      parental_leave__passes_12_month_employment_test: true
      parental_leave__applied_for_leave_or_stopped_working: true
      parental_leave__had_previous_parental_leave_in_last_six_months: true
```

So we test that when they have taken leave in the last six months, they are therefore not eligible.

Also, a test for when they're not a nz_citizen
and when they've not passed the 12 month employment test
etc
etc